front/public/locales/en/operationalStudies/manageTrainSchedule.json

```diff
@@ -60,6 +60,8 @@
   "maximumRunTime": "Maximum running time",
   "maximumRunTimeError": "Travel time must not exceed 12h",
   "No path could be found": "No path found",
+  "Pathfinding timed out: no path could be found": "Time limit exceeded, no path found",
+  "Relaxed pathfinding timed out: no path could be found, then relaxed path search timed out": "No path found, then exceeded time limit while searching unconstrained path",
```

Comment:
I didn't know we couldn't/weren't using keys for error matching in the front end (here, we do send `no_path_found:relaxed_path_timeout`), and instead use the entire string message sent from the back. I thought that was the plan. I'm a bit sad.

Reply:
That's weird, it does work with error codes in other parts. Especially here: https://github.com/OpenRailAssociation/osrd/blob/dev/front/public/locales/en/errors.json#L11-L14

I guess it's not related to this specific PR, but something is off here

Reply:
To be discussed (probably with @sim51) but after digging a bit on that, this translation is probably not used (could trigger the timeout, but not translate it in front).

Here are some responses obtained from `api/v2/infra/<x>/pathfinding/blocks`:
```json
{"status":"pathfinding_failed","core_error":{"status":500,"type":"core:no_path_found:timeout","context":{},"message":"Pathfinding timed out: no path could be found"}}
{"status":"pathfinding_failed","core_error":{"status":500,"type":"core:no_path_found:relaxed_path_timeout","context":{},"message":"Relaxed pathfinding timed out: no path could be found, then relaxed path search timed out"}}
```
What's displayed in Operational studies PF is `An error occurred in pathfinding: No path found.` (possibly translated in 🇫🇷 if kanguage configured).

Those error subtypes are useful for debugging and batch on endpoint directly, not for front (this may be OK to let it like that).

So my proposition would be:
* remove those lines (maybe also the one above)
* not dig more on this now, but maybe delegate it to another issue depending on the goal (https://github.com/OpenRailAssociation/osrd/issues/6685 ?)

Reply:
The whole editoast error system should be rewritten anyway (necessary for the crate split).

As for the matching on strings and not keys, after a little digging through the git history I'm pretty sure it's just an historical remnant from darker times ;)

Reply:
Keys are used for errors defined in editoast. But here, we are with a core error which is wrapped into an editoast error (moreover for editoast, core's errors are dynamics).  
That's why we can't use the same pipe as the one defined for editoast. 
Definitively, we should do something about that on the frontend, but it's a generic task about how to handle core's error on the frontend

Reply:
After discussion with @sim51 & @SharglutDev:
* removed the outdated translations: https://github.com/OpenRailAssociation/osrd/pull/7953/commits/f62952842dad7d5e0a5b10c8db9be227a61ed668
* created a dedicated issue for the handling of those errors: https://github.com/OpenRailAssociation/osrd/issues/8246 (please fill in more info if relevant)

So I'm resolving the (and will merge to the dedicated branch probably by noon): please don't hesitate to ask for more as this is not yet into `dev` there is some time to improve.